Truth or Truth/Screens/Settings/SettingsItem.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SettingsItem.swift
+//  Truth or Truth
+//
+//  Created by eli dangerfield on 8/3/20.
+//  Copyright © 2020 Josiah Rininger. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum SettingsItem: CaseIterable {
+    case About
+    case ReportABug
+    
+    func getIcon() -> UIImage? {
```

Comment:
`var icon: UIImage? {`
Typically when functions return something without taking in a parameter, are O(1) complexity, and can be named without a verb (and still be grokked) I prefer to use a variable